open_humans/api_views.py

```diff
@@ -125,14 +126,19 @@ def get(request):
 
                 sources[badge['label']].append(user['username'])
 
+        source_filter = request.GET.get('source')
+        source_labels_and_names = [
+            (source_filter, app_label_to_verbose_name(source_filter)),
```

Comment:
sorry for the complexity in the backend here! some of the sources are "projects" and others are "apps" – this function only works for apps. (fwiw "apps" is deprecated :sweat:)

(I have a rewrite ready that avoids this and another issue – just leaving notes :) )